app/src/test/java/com/pmatuki/wowpapers/usecases/GetWallpapersTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.pmatuki.wowpapers.usecases
+
+import com.pmatuki.wowpapers.data.WallpapersRepositoryImpl
+import com.pmatuki.wowpapers.fake.WallpaperListSourceFake
+import com.pmatuki.wowpapers.usecases.get.GetWallpapers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Assert
+import org.junit.Test
+
+
+class GetWallpapersTest {
+
+    private val wallpaperListSourceFake = WallpaperListSourceFake(WALLPAPER_COUNT)
+
+    private val getWallpapers = GetWallpapers(WallpapersRepositoryImpl(wallpaperListSourceFake))
+
+    @ExperimentalCoroutinesApi
+    @Test
+    fun `test if get wallpapers use case returns expected list`()  = runBlockingTest {
```

Comment:
Like a comment above, maybe in this test is a good idea to validate different states like possible errors, empty lists and any other corner case this API may return.